src/test/java/org/folio/rest/support/ResponseUtilTest.java

```diff
@@ -0,0 +1,75 @@
+package org.folio.rest.support;
+
+import static org.hamcrest.CoreMatchers.instanceOf;
+import static org.hamcrest.CoreMatchers.is;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertNull;
+import static org.junit.Assert.assertThat;
+import static org.junit.Assert.assertTrue;
+
+import java.lang.reflect.Constructor;
+
+import javax.ws.rs.core.Response;
+
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+public class ResponseUtilTest {
+  @Rule
+  public ExpectedException expectedException = ExpectedException.none();
+
+  @Test
+  public void canReturnFalseIfNullResponse() {
+    assertFalse(ResponseUtil.hasCreatedStatus(null));
+  }
+
+  @Test
+  public void canReturnTrueIfCreatedStatus() {
+    Response response = Response.status(Response.Status.CREATED).build();
+
+    assertTrue(ResponseUtil.hasCreatedStatus(response));
+  }
+
+  @Test
+  public void canReturnFalseIfNotCreatedStatus() {
+    Response response = Response.status(Response.Status.ACCEPTED).build();
+
+    assertFalse(ResponseUtil.hasCreatedStatus(response));
+  }
+
+  @Test
+  public void canNotInstantiateClass() throws Exception {
```

Comment:
Can this unit test been extended and replaced by UtilityClassTester.assertUtilityClass? See StorageHelperTest for an example.

Reply:
@julianladisch, thanks, it really useful.